src/main/scala/com/gu/contentapi/models/AcastLog.scala

```diff
@@ -178,8 +178,10 @@ object AcastLog {
     decodingMap.foldLeft(encoded) { case (cur, (from, to)) => cur.replaceAll(from, to) }
   }
 
-  /* filter out partial content requests that are not starting from 0 byte */
-  val onlyDownloads: AcastLog => Boolean = { log => log.status != "206" || log.range.startsWith("0-") }
+  /* filter out partial content requests unless the byte-range starts from 0 and is not 0-1 */
```

Comment:
I can't help but feel that the regular expression is a bit overkill for this requirement. I think that it would be easier to read and verify the expression if it was `|| (log.range.startsWith("0-") && log.range != "0-1")`.

Reply:
:+1:

Reply:
Code will be easier to read and will almost certainly be faster